api/v3/openapi.yaml

```diff
@@ -1011,15 +1011,27 @@ components:
       type: object
       required:
         - topic_name
-        - partitions_count
-        - replication_factor
       properties:
         topic_name:
           type: string
         partitions_count:
           type: integer
         replication_factor:
           type: integer
+        replicas_assignments:
```

Comment:
nit: replica_assignments

Reply:
Note that I've been using this somewhat cumbersome wording everywhere - that's because that's the wording `NewTopic` uses. I'd prefer to keep it as-is - I did notice that such a convention is not present or enforced (e.g. you're using `partitions_count`/`partitionsCount` and not `numPartitions`), but for a hard to name field like this it's the easy to pick choice.